### PR TITLE
Fix storybook + tailwind setup

### PR DESCRIPTION
**Pull request by bxav**

### CODE REVIEW

**Comment by bxav** [comment]
Inspired by https://github.com/vercel/turbo/tree/main/examples/with-tailwind, https://storybook.js.org/blog/get-started-with-storybook-and-next-js/ and https://theodorusclarence.com/blog/nextjs-storybook-tailwind

File: packages/ui/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import * as React from "react";
```
Comment (GabinH) [inline packages/ui/src/index.tsx:1]: With React v18, no more needs to import React

File: apps/web/package.json [hunk 2]
```diff
@@ -19,6 +19,8 @@
     "eslint": "7.32.0",
     "eslint-config-custom": "*",
     "tsconfig": "*",
+    "tailwind-config": "*",
+    "tailwindcss": "^3.2.4",
```
Comment (GabinH) [inline apps/web/package.json:23]: Is not postcss required with tailwind ? (as you create a postcss config in this repo)

File: packages/tailwind-config/tailwind.config.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const colors = require("tailwindcss/colors");
+
+module.exports = {
+  content: [
+    // app content
+    `src/**/*.{js,ts,jsx,tsx}`,
+    // include packages if not transpiling
+    // "../../packages/**/*.{js,ts,jsx,tsx}",
```
Comment (GabinH) [inline packages/tailwind-config/tailwind.config.js:8]: comments to clean if not necessary

File: apps/story/stories/Button.stories.tsx [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 
-import { Button } from './Button';
+import { Button } from 'ui';
```
Comment (GabinH) [inline apps/story/stories/Button.stories.tsx:4]: Can be good to rename each packages and apps name with @nextwork prefix no ?

Comment (bxav) [reply apps/story/stories/Button.stories.tsx:4]: Yes, I think it's a good practice for the project in question. But not in the scope here for this template.

File: apps/web/package.json (hunk 2 above)
Comment (bxav) [reply apps/web/package.json:23]: Good catch, that's why I had the hotreload issue

**Pull request merged**